app/Models/Department.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Department extends Model
```

Comment:
Toàn bộ model chưa hề có softdelete

Reply:
đã thêm $table->softdelete() vào tất cả các bảng